src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx

```diff
@@ -48,28 +49,50 @@ export const DeploySuccess = ({ projectId }: { projectId: number }) => {
 
   return (
     <div className="mt-4 flex flex-col items-center justify-center text-center">
-      <Image src="/assets/images/dancing.gif" width={218} height={218} />
-      <div className="pt-8 text-4xl font-medium">Congratulations!</div>
-      <div className="mt-6 text-xl font-normal">{deployGreeting}</div>
-      <div className="flex flex-col gap-2 py-12">
-        <Button
-          type="primary"
-          onClick={handleGoToProject}
-          loading={gotoProjectClicked}
-        >
-          <span>
-            <Trans>Go to project</Trans>
-          </span>
-        </Button>
+      <Image
+        src="/assets/images/create-success-hero.webp"
+        width={380}
+        height={371}
+      />
+      <div className="pt-8 font-display text-5xl font-bold">
+        <Trans>Congratulations!</Trans>
+      </div>
+      <div className="mt-4 text-lg font-normal text-grey-600 dark:text-slate-200">
+        {deployGreeting}
+      </div>
+      <XLButton
+        type="primary"
+        onClick={handleGoToProject}
+        loading={gotoProjectClicked}
+        className="mt-8"
+      >
+        <span>
+          <div className="flex items-center gap-3 text-white">
```

Comment:
**issue** cant have div within span, there'll be a console error

Reply:
Ah yep, fixed and still all works:

<img width="1013" alt="Screen Shot 2023-05-05 at 9 50 19 am" src="https://user-images.githubusercontent.com/96150256/236354689-ceb9b196-dcfa-449b-b10a-af6beb659b9a.png">